cms/articles/models.py

```diff
@@ -432,6 +433,11 @@ def dataset_document_list(self) -> list[dict[str, Any]]:
             dataset_documents = sorted(dataset_documents, key=lambda d: d["title"]["text"])
         return dataset_documents
 
+    @cached_property
+    def parent_for_choosers(self) -> Page:
+        topic_page_class = resolve_model_string("topics.TopicPage")
+        return topic_page_class.objects.ancestor_of(self).first().specific_deferred
+
```

Comment:
```suggestion
    def parent_for_choosers(self) -> Page:
        """Display the ancestor Topic page instead of the Article Series page in choosers."""
        topic_page_class = resolve_model_string("topics.TopicPage")
        return topic_page_class.objects.ancestor_of(self).first().specific_deferred

```
😇

Reply:
49b3cfee